src/ontodia/data/sparql/sparqlDataProviderSettings.ts

```diff
@@ -93,7 +107,44 @@ export interface FullTextSearchSettings {
     extractLabel?: boolean;
 }
 
-export const WikidataSettings: SparqlDataProviderSettings = {
+export const RDFSettings: SparqlDataProviderSettings = {
+    refElementLinkQueryOut: '\${refElementIRI} \${refElementLinkIRI} ?inst . FILTER ISIRI(?inst)',
+    refElementLinkQueryIn: '?inst \${refElementLinkIRI} \${refElementIRI}. ',
+    refElementQueryOut: '\${refElementIRI} ?link ?inst . FILTER ISIRI(?inst)',
+    refElementQueryIn: '?inst ?link \${refElementIRI} .',
+
+    linksInfoQuery: `SELECT ?source ?type ?target
+            WHERE {
+                ?source ?type ?target.
+                VALUES (?source) {\${ids}}
+                VALUES (?target) {\${ids}}                
+            }`,
+
+    defaultPrefix: ``,
```

Comment:
Don't use template literals when it's just a single-line string with no template substitutions.